src/components/ImageCard/HotspotContent.test.jsx

```diff
@@ -122,7 +122,7 @@ describe('HotspotContent', () => {
       />
     );
     // threshold should be found and french formatted
-    expect(getAllByTitle(/temperature > 0,05/)).toHaveLength(2);
```

Comment:
Why did this change, and could we write the tests differently so that it wouldn't have to change?

Reply:
`<Icon/>` always renders a `<title>` in the `<svg>`. So the `svg` and the `button` were both getting a title. 

Removing the unnecessary title in the `svg` is why it moved down to expect 1. Ideally we'd use [`getByRole` instead](https://testing-library.com/docs/guide-which-query), but here I wanted to preserve the test as is as much as possible.